pom.xml

```diff
@@ -100,7 +100,7 @@
       <dependency>
         <groupId>com.fasterxml.jackson.core</groupId>
         <artifactId>jackson-databind</artifactId>
-        <version>${version.jackson}.1</version>
```

Comment:
I have NO idea why the version is written like this.  How about we just move the .3 up to the version.jackson property.